includes/class-wc-google-analytics.php

```diff
@@ -321,7 +321,10 @@ protected function get_standard_tracking_code() {
 	 */
 	protected function get_ecommerce_tracking_code( $order_id ) {
 		// Get the order and output tracking code
-		$order = new WC_Order( $order_id );
+		$order = wc_get_order( $order_id );
+
+		// Make sure we have a valid order object
+		if ( ! $order || is_wp_error( $order ) ) return "";
```

Comment:
Thanks! Some notes on this. Errors are already handled for us so you can simplify to `if ( ! $order ) {...}` And mind coding standards such as period at the end of the comment and no inline condition statements like that. And use single quotes instead of double quotes.

Reply:
Thanks for the feedback. I'm happy to make all of those changes, but I wanted to double check on some things first.

1. If the `$order` variable was indeed an instance of WP_Error, wouldn't the simple `! $order` miss it? Or are you saying that `wc_get_order()` will never return a `WP_Error` object?

2. I tried to match the code style of the rest of the function regarding periods at the end of the comment and quote style. I can definitely change that, but then it wouldn't match what's there. Please confirm.

Reply:
Exception is handler up the chain and when caught will return a false to the caller. Yeah so the existing code may have been written long ago and possibly by different people so any new code we add it must follow the WP coding standards.